doc/cephfs/snapshots.rst

```diff
@@ -0,0 +1,81 @@
+================
+CephFS Snapshots
+================
+
+CephFS snapshots create an immutable view of the file system at the point
+in time they are taken. CephFS supports snapshots which is managed in a 
+special hidden subdirectory named ``.snap`` .Snapshots are created using
+``mkdir`` inside this directory.
+
+It can be exposed with a different name by setting the below client configurations.
```

Comment:
"It" should be changed to something more meaningful to the first-time reader. Probably this sentence should read "Snapshots can be exposed with a different name by changing the following client configurations:"